### PR TITLE
Improve FileBrowser UI

### LINKED ISSUE #714: Improve FileBrowserUI
* Show path as subtitle
* Rearrange texts, see long filenames
* group "shared to users" display for share to multiple users files

### LINKED ISSUE #724: Multi row filenames overlapp with filemetadata in filebrowser view
In the filebrowser some long filenames overlapp the meta data in the upper right corner like in this screenshot:
tmp_25536-screenshot_20161003-103749-1457220072

### PR DESCRIPTION
**Pull request by JoGir**
resolves #714
resolves #724 
* Show path
* fix layout for filenames
* fix layout for shared to multiple users label

depends on #731

### CODE REVIEW

**Comment by JoGir** [comment]
NullPointer ^^
fix in progress

File: qabelbox/src/main/java/de/qabel/qabelbox/box/dto/BoxPath.kt [hunk 1]
```diff
@@ -42,5 +41,8 @@ sealed class BoxPath() {
 
 
     fun toList(): List<String> = if (this is Root) emptyList() else parent.toList() + name
+
+    fun toReadable(): String = "/" + toList().joinToString("/")
```
Comment (julianseeger) [inline qabelbox/src/main/java/de/qabel/qabelbox/box/dto/BoxPath.kt]: This is diverging from https://github.com/Qabel/qabel-core/blob/90145780680b7954f53d068126ec883a5d6e20ff/box/src/main/java/de/qabel/box/storage/dto/BoxPath.kt
Don't know why we have two implementations (probably my bad). Can you switch android to use the core implementation? the BoxPath.toString() from the core implementation is meant to do this (`toReadable`)

Comment (JoGir) [inline qabelbox/src/main/java/de/qabel/qabelbox/box/dto/BoxPath.kt]: Yes i saw it, but prefer to create an extra PR to replace all box classes moved to core

Comment (JoGir) [inline qabelbox/src/main/java/de/qabel/qabelbox/box/dto/BoxPath.kt]: created issue for that #733. We should determine which other classes  needed core

**Comment by JoGir** [comment]
Layout
layout_file_item

**Comment by JoGir** [comment]
capture _2016-10-07-01-58-39
capture _2016-10-07-01-59-41

**Comment by JoGir** [comment]
Separator
capture _2016-10-07-02-31-08

**Comment by JoGir** [comment]
Smells good, tastes good and is green

**Comment by julianseeger** [comment]
*jummy*

**Pull request merged**